### DISCUSSION TITLE
Altering metadata

### DISCUSSION THREAD

**Discussion by SRawrKnight** [openaudible/openaudible#826] [Q&A]
If I alter the metadata of the files in my OpenAudible books folder, what will the result be? In the past I would usually change the tags & filename such that a series of books would be grouped together and put in order. I can see that OpenAudible probably wouldn't even be able to sort using those tags, but would it break the ability to keep those library items synced?

**Comment by Cfomodz** [comment]
**Yes, it will likely break synchronization.** OpenAudible requires consistent filenames across all file types (AAX, MP3, M4B, images) to maintain library connections.

## The Fix

**For organizing book series:**

1. **Use OpenAudible's rename feature** (right-click → Rename)
   - This renames all associated files consistently
   - Database updates automatically
   - Example pattern: `Author - Series 01 - Title`

2. **OR export to a separate library**
   - Let OpenAudible maintain its original folder structure
   - Copy exported files elsewhere for custom organization
   - Apply metadata changes in the copy, not OpenAudible's folder

## Critical Rules

- ❌ **DON'T** manually rename files outside OpenAudible
- ❌ **DON'T** modify Product ID or ASIN metadata fields
- ✅ **DO** use the built-in rename function
- ✅ **DO** keep OpenAudible updated (version 3.5.8+ recommended)
